client/ctx.go

```diff
@@ -0,0 +1,37 @@
+package client
+
+import (
+	"context"
+
+	"google.golang.org/grpc/metadata"
+)
+
+const (
+	logLevelRPCMetaKey = "log_level"
+	clientRequestIDKey = "client_request_id"
```

Comment:
It is recommended to define context key type other than using const string

Reply:
These are metadata keys for grpc context, it's required to use string as key

Reply:
ok